antefinance/ante-community-tests
# Arbitrum tests

Tests for Arbitrum One network

- AAVE 
    - total supply plunge test
- Abracadabra 
    - balance holder threshold
- Across
    - balance holder threshold
- Balancer
    - balance holder threshold
- Chainlink
    - check LINK/USD , ETH/USD feeds for daily updates 
    - check LINK/USD, ETH/USD feeds heartbeat
- DAI peg test
- FRAX peg test
- USDT peg test
- USDC peg test
